modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="alibaba-ai-converting-image-to-qcow2_{context}"]
```

Comment:
I recommend that you follow OCP doc conventions for naming modules. If you look at the other installation modules, you see that they begin with 'installation-'. This makes it easier to do global changes and to find files in the installation assemblies.

Reply:
When there are ten modules (like in the final version) it's very easy when everything starts with alibaba-ai, and I saw it done in other modules too. But I'll check this for the final version.

Reply:
The main problem with using a non-standard naming convention is that it is harder to find the files in the huge modules folder. Only someone who knows that 'alibaba-ai' is for Alibaba / Assisted Installer would find those files. 

On the other hand, I see lots of modules that begin with 'aws-' and belong to different guides, so maybe 'alibaba' is just fine. :-)